rotkehlchen/tests/unit/decoders/test_cowswap.py

```diff
@@ -1085,3 +1085,62 @@ def test_gnosis_eure_v2(
         counterparty=CPT_COWSWAP,
         address=GPV2_SETTLEMENT_ADDRESS,
     )]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x3Ba6eB0e4327B96aDe6D4f3b578724208a590CEF']])
```

Comment:
that looks off. The account is not a smart account. That's not testing what you changed.